### PR TITLE
Update helm v3 commands in README.md

### PR DESCRIPTION
**Pull request by pure-jliao**
Helm charts docs updated for helm v3.

### CODE REVIEW

File: pure-csi/README.md [hunk 1]
```diff
@@ -57,6 +57,13 @@ helm repo update
 helm search pure-csi
 ```
 
+Changes for helm v3
```
Comment (Pure-AdamuKaapan) [inline pure-csi/README.md]:
```suggestion
### Changes for Helm 3
```

File: pure-csi/README.md [hunk 2]
```diff
@@ -165,6 +172,13 @@ helm install --name pure-storage-driver pure/pure-csi --namespace <namespace> -f
             --set namespace.pure=k8s_xxx \
 ```
 
+Changes for helm v3
```
Comment (Pure-AdamuKaapan) [inline pure-csi/README.md]:
```suggestion
### Changes for Helm 3
```

File: pure-k8s-plugin/README.md [hunk 3]
```diff
@@ -35,6 +35,13 @@ helm repo update
 helm search pure-k8s-plugin
 ```
 
+Changes for helm v3
```
Comment (Pure-AdamuKaapan) [inline pure-k8s-plugin/README.md]:
```suggestion
### Changes for Helm 3
```

File: pure-k8s-plugin/README.md [hunk 4]
```diff
@@ -145,6 +152,13 @@ helm install --name pure-storage-driver pure/pure-k8s-plugin --namespace <namesp
             --set orchestrator.name=openshift
 ```
 
+Changes for helm v3
```
Comment (Pure-AdamuKaapan) [inline pure-k8s-plugin/README.md]:
```suggestion
### Changes for Helm 3
```

**Comment by pure-jliao** [comment]
> Looks pretty good, just had some quick nits for formatting (basically just made "Changes for helm v3" into a heading level 3, and then capitalized Helm and changed v3 to just "3").

Since following contents do not necessarily belong to this heading, would it be better just highlight them.

File: pure-csi/README.md [hunk 5]
```diff
@@ -165,6 +172,13 @@ helm install --name pure-storage-driver pure/pure-csi --namespace <namespace> -f
             --set namespace.pure=k8s_xxx \
 ```
 
+**Changes for Helm 3**
+
+```bash
+# --name flag is no longer needed, see usage below:
+helm install [NAME] [CHART] [flags]
```
Comment (trevormeiss) [inline pure-csi/README.md]: Can we put the full command so it's easier to copy-paste?
```suggestion
helm install pure-storage-driver pure/pure-csi --namespace <namespace> -f <your_own_dir>/yourvalues.yaml \
            --set flasharray.sanType=fc \
            --set namespace.pure=k8s_xxx \
```

File: pure-k8s-plugin/README.md [hunk 6]
```diff
@@ -145,6 +152,13 @@ helm install --name pure-storage-driver pure/pure-k8s-plugin --namespace <namesp
             --set orchestrator.name=openshift
 ```
 
+**Changes for Helm 3**
+
+```bash
+# --name flag is no longer needed, see usage below:
+helm install [NAME] [CHART] [flags]
```
Comment (trevormeiss) [inline pure-k8s-plugin/README.md]: Same comment here

**Comment by pure-jliao** [comment]
> Instead of "Changes for Helm 3", can we split it into two sections?
> Something like
> 
> ```
> ## How to install
> 
> ### Helm 2
> ...
> 
> ### Helm 3
> ...
> ```
> 
> I feel that may be easier to understand

@trevormeiss - Updated, I think adding `# Helm 3` into the code block might be better, so no need to copy & paste more code.

**Pull request merged**